azure-pipelines.yml

```diff
@@ -372,7 +370,7 @@ stages:
   - stage: vNextIATDeploy
     dependsOn:  Devdeploy
     displayName: "vNext IATdeploy (inc terraform, webapp deploy)"
-    condition: and(succeeded(), or(eq(variables['Build.SourceBranch'], 'refs/heads/main'),startsWith(variables['Build.SourceBranch'], 'refs/heads/release/'), eq(variables['Build.SourceBranch'], 'refs/heads/develop')))    
+    condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/develop'))
```

Comment:
Should this not be 'startswith'?

Reply:
As discussed , current changes suffice the above requirement.